Sources/inc/CosmicCORSIKA.hh

```diff
@@ -229,30 +234,35 @@ namespace mu2e {
       static constexpr float _GeV2MeV = CLHEP::GeV / CLHEP::MeV;
       static constexpr float _cm2mm = CLHEP::cm / CLHEP::mm;
       static constexpr float _ns2s = CLHEP::ns / CLHEP::s;
+      static constexpr unsigned _fbsize_words = 5733 + 2;
```

Comment:
A comment about 5733 + 2 would be helpful.

Reply:
Added, thanks.